drivers/eks/eks_driver.go

```diff
@@ -317,6 +325,7 @@ func getStateFromOptions(driverOptions *types.DriverOptions) (state, error) {
 	state.AMI = options.GetValueFromDriverOptions(driverOptions, types.StringType, "ami").(string)
 	state.AssociateWorkerNodePublicIP, _ = options.GetValueFromDriverOptions(driverOptions, types.BoolPointerType, "associate-worker-node-public-ip", "associateWorkerNodePublicIp").(*bool)
 	state.KeyPairName = options.GetValueFromDriverOptions(driverOptions, types.StringType, "keyPairName").(string)
+	state.EbsEncryption, _ = options.GetValueFromDriverOptions(driverOptions, types.BoolPointerType, "ebs-encryption", "EbsEncryption").(bool)
```

Comment:
Drop the `_`, let's try and keep the formatting the same with all the other options.